components/kyma-environment-broker/internal/process/provisioning/knative_natss_step.go

```diff
@@ -0,0 +1,42 @@
+package provisioning
```

Comment:
I'd recommend to have one single step for natss-streaming and knative-provisioner-natss. As they are functionally one step from provisioning point of view.

Reply:
For the future time without knative, we will not need "knative-provisioner-natss" anymore. I suppose that it will be easier to remove it.

Reply:
yea, but we will not even have natss as well for the managed offering ;)

Reply:
You mean that for "trial" we will go with EMS, right? And NATSS will remain only on open source.